### ISSUE TITLE
Exporting to events file set all angles to 180

### ISSUE THREAD

**Issue by openface** [InclementDab/DayZ-Editor#91]
After placing several objects down and positioning/rotating them to the correct location, I exported the configuration to events file, which gave me positions but the angles are all set to 180.  Seems like a bug, so am filing it here.

PS. Thanks for this awesome tool.
<3

Full exported events file:

```
<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
<events>
	<event name="Land_Wreck_Caravan_MGreen">
		<pos x="5637.46" y="255.142" z="9210.54" a="180"/>
	</event>
	<event name="Land_Wreck_Caravan_MGreen">
		<pos x="6816.31" y="286.772" z="9382.78" a="180"/>
	</event>
	<event name="Land_Wreck_Caravan_MGreen">
		<pos x="7430.81" y="220.383" z="8848.6" a="180"/>
	</event>
	<event name="Land_Wreck_Caravan_MGreen">
		<pos x="8111.02" y="228.692" z="8663.38" a="180"/>
	</event>
	<event name="Land_Wreck_Caravan_MGreen">
		<pos x="8611.1" y="240.175" z="8334.58" a="180"/>
	</event>
	<event name="Land_Wreck_Caravan_MGreen">
		<pos x="9997.76" y="141.673" z="7846.17" a="180"/>
	</event>
	<event name="Land_Wreck_Caravan_MGreen">
		<pos x="10548.4" y="108.455" z="7739.97" a="180"/>
	</event>
	<event name="Land_Wreck_Caravan_MGreen">
		<pos x="11657.5" y="82.6484" z="7990.82" a="180"/>
	</event>
	<event name="Land_Wreck_Caravan_MGreen">
		<pos x="11828.1" y="85.0963" z="8378.52" a="180"/>
	</event>
	<event name="Land_Wreck_Caravan_MGreen">
		<pos x="8926.87" y="226.585" z="8047.37" a="180"/>
	</event>
	<event name="Land_Wreck_Caravan_MGreen">
		<pos x="9212.08" y="211.981" z="7671.3" a="180"/>
	</event>
	<event name="Land_Wreck_Caravan_MGreen">
		<pos x="8825.91" y="126.521" z="6553.4" a="180"/>
	</event>
	<event name="Land_Wreck_Caravan_MGreen">
		<pos x="9220.67" y="124.424" z="6539.12" a="180"/>
	</event>
	<event name="Land_Wreck_Caravan_MGreen">
		<pos x="9583.21" y="103.358" z="6272.52" a="180"/>
	</event>
	<event name="Land_Wreck_Caravan_MGreen">
		<pos x="9978.48" y="92.6529" z="6113.57" a="180"/>
	</event>
	<event name="Land_Wreck_Caravan_MGreen">
		<pos x="11483.6" y="92.6396" z="5317.57" a="180"/>
	</event>
	<event name="Land_Wreck_Caravan_MGreen">
		<pos x="11569.4" y="138.306" z="6302.39" a="180"/>
	</event>
	<event name="Land_Wreck_Caravan_MGreen">
		<pos x="11312.6" y="99.5656" z="7102.35" a="180"/>
	</event>
	<event name="Land_Wreck_Caravan_MGreen">
		<pos x="10163.2" y="103.863" z="9541.66" a="180"/>
	</event>
	<event name="Land_Wreck_Caravan_MGreen">
		<pos x="9731.96" y="245.479" z="8746.97" a="180"/>
	</event>
	<event name="Land_Wreck_Caravan_MGreen">
		<pos x="6859.18" y="94.3458" z="5972.66" a="180"/>
	</event>
	<event name="Land_Wreck_Caravan_MGreen">
		<pos x="4862.81" y="172.121" z="7854.76" a="180"/>
	</event>
	<event name="Land_Wreck_Caravan_MGreen">
		<pos x="5294.07" y="206.229" z="8176.94" a="180"/>
	</event>
	<event name="Land_Wreck_Caravan_MGreen">
		<pos x="5156.48" y="146.484" z="7200.18" a="180"/>
	</event>
	<event name="Land_Wreck_Caravan_MGreen">
		<pos x="6214.91" y="197.008" z="6664.08" a="180"/>
	</event>
	<event name="Land_Wreck_Caravan_MGreen">
		<pos x="5618.84" y="130.769" z="6870.96" a="180"/>
	</event>
	<event name="Land_Wreck_Caravan_MGreen">
		<pos x="6216.67" y="124.214" z="6323.2" a="180"/>
	</event>
	<event name="Land_Wreck_Caravan_MGreen">
		<pos x="6478.28" y="125.883" z="4800.74" a="180"/>
	</event>
	<event name="Land_Wreck_Caravan_MGreen">
		<pos x="8141.92" y="83.3431" z="5104.58" a="180"/>
	</event>
</events>
```

**Comment by InclementDab** [comment]
Going to attempt to resolve this. Thanks for the report

**Comment by InclementDab** [comment]
Fixed in version 1.2
